### DISCUSSION TITLE
How to use Osmand with Android Auto?

### DISCUSSION THREAD

**Discussion by michaelbr51** [osmandapp/OsmAnd#18931] [General]
I was told that Android Auto is from Google, therefore is tied to GoogleMap, it seems it's possible to use Osmand through Android Auto. Can someone confirm this and explain me how to do it? I read that the only apps allows use of 3rd party navigation is AutoMate, but my car computer does not recognize it (in setting of car pc, shows only AA).
AutoMate 2.3.2
Android 10
car: Opel Insignia

**Comment by DmitryAlexei** [comment]
Please see docs about OsmAnd and Android Auto https://osmand.net/docs/user/navigation/auto-car
Android Auto is available for Google play apps: OsmAnd + version and OsmAnd Pro subscription. For more information, see https://osmand.net/docs/user/purchases/android#free-and-paid-features
